mopidy_tunein/tunein.py

```diff
@@ -176,10 +176,11 @@ def find_playlist_parser(extension, content_type):
 class TuneIn(object):
     """Wrapper for the TuneIn API."""
 
-    def __init__(self, timeout, session=None):
+    def __init__(self, timeout, filter=None, session=None):
         self._base_uri = 'http://opml.radiotime.com/%s'
         self._session = session or requests.Session()
         self._timeout = timeout / 1000.0
+        self._filter = filter
```

Comment:
What about creating the filter string here once rather than every time in `_tunein()`?